src/app/pages/server-listing/server-listing.component.ts

```
@@ -202,12 +203,16 @@ export class ServerListingComponent implements OnInit, OnDestroy {
         this.limit = parseInt(query.limit);
       }
 
-      const tableColumnsStr = this.storageHandler.get('serverListTableColumns');
-      if(tableColumnsStr) {
-        const tableColumns: string[] = JSON.parse(tableColumnsStr);
-        this.possibleColumns.forEach((column) => {
-          column.show = tableColumns.findIndex((item) => item === column.name) !== -1;
-        });
+      const tableColumnsStr = query.columns;
+      if(tableColumnsStr && tableColumnsStr.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)